mlf_core/util/rich.py

```diff
@@ -0,0 +1,18 @@
+import sys
+import os
+from rich.console import Console
+
+
+def force_terminal_in_github_action() -> Console:
+    """
+    Check, whether the GITHUB_ACTIONS environment variable is set or not.
+    If set, the process runs in a workflow file and we need to tell rich, in order to get colored output as well.
+    """
+    if 'GITHUB_ACTIONS' in os.environ:
+        return Console(file=sys.stderr, force_terminal=True)
+    else:
+        return Console(file=sys.stderr)
```

Comment:
I don't think that we want to print to stderr by default?

`console = rich.console.Console()`

this is what it was earlier